lunatrace/bsl/frontend/src/pages/project/settings/PrFeedback.tsx

```diff
@@ -53,18 +56,32 @@ export const SettingsPrComments: React.FC<SettingsPRCommentsProps> = ({ project
         </Col>
       </Row>
       <Row>
-        <Form className="d-flex justify-content-center">
-          <Form.Switch
-            className="form-switch-md"
-            id="pr-reviews-switch"
-            label={checked ? 'PR Comments Enabled' : 'PR Comments Disabled'}
-            checked={checked}
-            disabled={isLoading}
-            onChange={(e) => {
-              setChecked(!checked);
-              void updateSettings({ id: project.settings_id, settings: { pr_feedback_disabled: !e.target.checked } });
-            }}
-          />
+        <Form className="d-flex align-items-center flex-column">
```

Comment:
Oh and I merged this settings UI from these two components because it wasn't DRY and it didn't make too much sense how it was laid out in the UI.  Both the settings really belong in the same section under github feedback, so now they are